Runtime/Core/SelectionPickerRendererStandard.cs

```diff
@@ -0,0 +1,109 @@
+﻿using UObject = UnityEngine.Object;
+
+namespace UnityEngine.ProBuilder
+{
+    internal partial class SelectionPickerRenderer
+    {
+        internal class SelectionPickerRendererStandard: ISelectionPickerRenderer
+        {
+            /// <summary>
+            /// Render the camera with a replacement shader and return the resulting image.
+            /// RenderTexture is always initialized with no gamma conversion (RenderTextureReadWrite.Linear)
+            /// </summary>
+            /// <param name="camera"></param>
+            /// <param name="shader"></param>
+            /// <param name="tag"></param>
+            /// <param name="width"></param>
+            /// <param name="height"></param>
+            /// <returns></returns>
+            public Texture2D RenderLookupTexture(
+                Camera camera,
+                Shader shader,
+                string tag,
+                int width = -1,
+                int height = -1)
+            {
+                bool autoSize = width < 0 || height < 0;
+
+                int _width = autoSize ? (int)camera.pixelRect.width : width;
+                int _height = autoSize ? (int)camera.pixelRect.height : height;
+
+                GameObject go = new GameObject();
+                Camera renderCam = go.AddComponent<Camera>();
+                renderCam.CopyFrom(camera);
```

Comment:
There's a lot of camera setup that looks like it could be abstracted between all picking renderers, would it make sense to do so?

Reply:
I'm not sure, setup could evolve differently between pipelines that I chose to put a hard line rather than using a common base. I prefer to keep things separate in case we have to deal we specifics field/setup/data/etc..